### ISSUE TITLE
[BUG] Accessibility - App & Web App Videos that play automatically need to have a mechanism to pause playback

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#4863]
**Describe the issue**
The video that plays during Orientation screens (and any other automatically playing video) need to have a mechanism to pause or stop the playback. When using TalkBack the video opens and plays automatically and the user is unable to understand what TalkBack is saying. 

This shouldn't play automatically, but if it MUST, then it would be a more accessible experience to have the Screen readers focus land on the Play/Pause "Onboarding video, double tap to activate." Some users may be very literal and not understand "double tap to activate" as "Play/Pause video" I would suggest changing the label on this to indicate such. 

There is currently a way to "Skip" which stops the playback, but it is last in the tab order for screen reader users and should probably be fourth and the CC control third. 

The WCAG (Web Content Accessibility Guidelines) Success Criterion 2.2.2, known as Pause, Stop, Hide, is designed to help users control content that moves, blinks, scrolls, or auto-updates. This is particularly important for users with cognitive disabilities or attention deficits who may find such content distracting.

Here are the key points:

Moving, blinking, scrolling content: If content starts automatically, lasts more than five seconds, and is presented alongside other content, there must be a way for users to pause, stop, or hide it unless it is essential for the activity1.
Auto-updating content: If content updates automatically and is presented alongside other content, users must be able to pause, stop, hide it, or control the update frequency unless it is essential for the activity. 

**To Reproduce**
**Steps to reproduce the behavior:**
1. Go to newly downloaded app and launch Onboarding screens
2. Click on 'Continue' until user reaches video screen.  

**Expected behavior**
"Back button" > "Double tap to Play/Pause Onboarding video." > CC button > Skip

This shouldn't play automatically, but if it MUST, then it would be a more accessible experience to have the Screen readers focus land on the Play/Pause "Onboarding video, double tap to activate." Some users may be very literal and not understand "double tap to activate" as "Play/Pause video" I would suggest changing the label on this to indicate such. 

There is currently a way to "Skip" which stops the playback, but it is last in the tab order for screen reader users and should probably be fourth and the CC control third. 

**Actual behavior**
"Back button" > "Play/Pause" button "Onboarding video, double tap to activate." > Skip > CC button

When using TalkBack the video opens and plays automatically and the user is unable to understand what TalkBack is saying. This shouldn't play automatically, but if it MUST, then it would be a more accessible experience to have the Screen readers focus land on the Play/Pause "Onboarding video, double tap to activate."  

**Screenshots**

![Image](https://github.com/user-attachments/assets/fac5b82f-020a-41a6-a50d-5a85d1caf711)

**Testing Environment & Devices (please complete the following information):**
Device: Google Pixel 8a Android 14
Talkback 15.2

**Additional context**
This is also occurring on https://api.rokwire.illinois.edu/illinois/ However there is an additional bug there as the Play/Pause control is not working at all on web.

**Comment by acovher2** [comment]
https://github.com/user-attachments/assets/6f6b836d-7bc3-4f91-80b8-d574c6edf6db
Adding an example with audio to show how overwhelming auto-play can be to a Screen Reader User.

**Comment by acovher2** [comment]
Following the 3/11 meeting with JP and the team, I am updating the way these tickets are categorized to ensure they are placed in the standard bug queue and addressed accordingly. These are legitimate issues that impact users with disabilities, and as critical blockers, they should receive the same level of urgency as other high-priority bugs to ensure equitable access to the application for all users.

**Comment by acovher2** [comment]
Verified as still existing in App version 7.0.15 and web version 7.0.5 (yes, these are out of sync and I'm not sure they were supposed to be:
UIUC 7.0 Dev Web
https://api-dev.rokwire.illinois.edu/illinois/

**Comment by vburgett** [comment]
@TodorBachvarov @acovher2 to avoid the video automatically playing, could we have the video embedded in the first onboarding screen like the below screenshot? 

![Image](https://github.com/user-attachments/assets/34ea9400-1f04-4821-bb56-7208480e81a1)

**Comment by TodorBachvarov** [comment]
Hi @vburgett, I've added new Thumbnail button into the Get Started Panel. I use the video.thumb url, but if you want we can use some other image (just provide me the image resource). Tapping the button leads to the Video Tutorial Panel.
@acovher2, I've made the following improvements into the Video Tutorial Panel: 
1. Introduced new button label: "Onboarding Video Tutorial" hint: "double tap to play video". Tapping the button will open the Video Panel.
1. Video Content is marked as container and it focused immediately after video loading is finished.
2. I've added hint to the Video content: "Double tap to Play/Pause". 
3. Double tapping the Video content Play/Pause the video. 
With the following changes, when the User land on the Video tutorial button first, and the hint will let him know that the video will be played. After opening the Video Tutorial Panel the focus will be moved to the Video. In this way the only required action from the user to pause the video will be to double tap.

**Comment by acovher2** [comment]
I have verified this is fixed in the web app and the #4864 and #4873 cover remaining accessibility issues. 
Closing.
